backend/src/main/java/com/google/capstone/ServeUploads.java

```diff
@@ -0,0 +1,58 @@
+package com.google.capstone;
+ 
+import com.google.capstone.UploadedImage;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import com.google.gson.Gson;
+
+import java.io.IOException; 
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that gets all stored entities from Datastore
+ * Turns entities to UplaodedImage objects and converts to json
+ * then returns the json in response
+ */
+@WebServlet("/api/v1/serveUploads")
+public class ServeUploads extends HttpServlet {
```

Comment:
Maybe rename to getUploads to reduce confusion with serving like ServeBlob does?